client/app/js/controllers/wizard.js

```diff
@@ -1,7 +1,7 @@
 GLClient.controller('WizardCtrl', ['$scope', '$rootScope', '$location', '$route', '$http', '$uibModal', 'Admin',
-                    'DefaultAppdata', 'CONSTANTS',
+                    'DefaultAppdata', 'GLTranslate', 'CONSTANTS',
```

Comment:
instead of passing it to all controllers that would need it probably we can inject it in the MainCtrl only, this way it will be inherited in the scope of all the other.